verifier-sdk/src/light_transaction.rs

```diff
@@ -601,6 +588,31 @@ impl<
         Ok(())
     }
 
+    /// Returns a Transaction Merkle Tree which should be used for the current
+    /// transaction. It might be either:
+    ///
+    /// * A new Transaction Merkle Tree provided as a remaining account, which
+    ///   usually is the case when we reached the switch threshold (255_000).
```

Comment:
```suggestion
    ///   usually is the case when we reached the switch threshold (255_000).
    ///   We pass a new Transaction Merkle Tree as remaining account because Anchor does not
    ///   support passing two accounts of the same type in the same instruction.
    ///   We need a second Transacton Merkle Tree account for the utxos we want to spend
    ///   in the transaction are in the old Transaction Merkle Tree.
    ///   Since the old Merkle Tree is almost full we need to insert
    ///   the new utxo commitments into the new Transaction Merkle Tree.
```